src/Commands/Backup/Automatic/DisableCommand.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Backup\Automatic;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+
+/**
+ * Class CancelCommand
+ * @package Pantheon\Terminus\Commands\Backup\Automatic
+ */
+class DisableCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+
+    /**
+     * Disable a regular backup schedule
+     *
+     * @authorize
+     *
+     * @command backup:automatic:disable
```

Comment:
I'd like to note (for future discussion) that it's odd that in some places we use the full word (e.g. `automatic`) but in others steadfastly use the short (e.g. `env`, `org`).